src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -350,7 +350,25 @@ public function filter_template_include( $template ) {
 			return $template;
 		}
 
-		$context   = tribe_context();
+		// Grab the context early so folks can use it if they bypass our templates.
+		$context = tribe_context();
+
+		/**
+		 * Allows filtering the loading of our proprietary templates.
+		 *
+		 * @since TBD
+		 *
+		 * @param boolean        $load     Whether we should load the theme templates instead of the Tribe templates. Default false.
+		 * @param string         $template The template located by WordPress.
+		 * @param Tribe__Context $context  The singleton, immutable, global object instance.
+		 */
+		$load_template = apply_filters( 'tribe_events_views_v2_should_load_default_templates', false, $template, $context );
```

Comment:
Move this at the top of the function: no point in doing a bunch of other costly checks if this filter is already returning `false`. If this filter is returning `true`, then apply our logic.